src/Analysis/GWASResultsV2/Utils/SharedContext.jsx

```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react';
+
+const SharedContext = createContext();
+
+export default SharedContext;
```

Comment:
minor comment: do we need an extra file for this? Or would it also work if we have this as part of GWASResultsContainer?

Reply:
I don't think this would work without an extra file or adding it as an export in GWASResultsContainer, which I'd prefer to avoid.